### PR TITLE
Add Image to Expandable Text Widget

### PR DESCRIPTION
**Pull request by eunicePops**
https://user-images.githubusercontent.com/40694423/137401706-8937beb7-e5eb-4f17-8540-913489daae59.mp4

https://user-images.githubusercontent.com/40694423/137002717-8f410260-52c7-45a6-aa8e-22c564565055.mp4

https://user-images.githubusercontent.com/40694423/137096625-abf85b89-08ed-4ddf-b06f-f8933914c868.mp4

### CODE REVIEW

File: lib/ui/view/expandable_textfield/expandable_textfield_screen.dart [hunk 1]
```diff
@@ -24,7 +26,7 @@ class ExpandableTextFieldScreen extends HookWidget {
       required this.channelID})
       : super(key: key);
   final Widget widget;
-  final Function(String message) sendMessage;
+  final Function(String message, List<File> media) sendMessage;
```
Comment (jeremiahseun) [inline lib/ui/view/expandable_textfield/expandable_textfield_screen.dart:29]: media should be optional and not a required value

File: lib/ui/view/expandable_textfield/expandable_textfield_screen.dart [hunk 2]
```diff
@@ -224,8 +227,8 @@ class ExpandableTextFieldScreen extends HookWidget {
                                               if (textController.text
                                                   .toString()
                                                   .isNotEmpty) {
-                                                sendMessage(
-                                                    textController.text);
+                                                sendMessage(textController.text,
+                                                    model.mediaList);
```
Comment (jeremiahseun) [inline lib/ui/view/expandable_textfield/expandable_textfield_screen.dart:231]: Please make media optional.
What you did means I must send media with text which should not be so

Comment (eunicePops) [reply lib/ui/view/expandable_textfield/expandable_textfield_screen.dart:231]: changes made

**Review by pauleke65 — DISMISSED** [review]
I don't really get what the task done here.
In the first video you just opened the gallery, and in the second you just sent a message.

There's no prove of media upload, and the way it was done in code is wrong.
Typically you upload the file first the get it's URL, that's when you now send the URL of the file as a string with the message.
Refer to the Channels API docs for more

**Comment by eunicePops** [comment]
Made changes and trying to test out the code but channels are not
displaying, trying to create new channel is throwing this.


Reached out to backend

**Review by jeremiahseun — APPROVED** [review]
LGTM

**Review by YenumJay — APPROVED** [review]
LG

**Pull request merged**